terraform/jupyterlab-ibmcloud/scripts/install_cuda.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+# Install cuda driver and libraries 
```

Comment:
CUDA; and would it make sense to have this get the latest version, or don't they have convenient links for the 'newest' driver?  Or was there a reason you needed this version?  Just thinking of trying to keep it 'current' over time with minimal updates.

Reply:
I am running tensorflow 1.15 in the Jupyter Lab and wanted to stick with a known workin Cuda. I agree, needs to be less costly in updates. Will add a todo to make it more resilient to cuda library changes for now.